PeerioTechnologies/peerio-desktop
# feat: Search folders when moving

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### PR author

#### Relevant issue tracker links  
https://peerio.tpondemand.com/RestUI/Board.aspx#page=task/7636&appConfig=eyJhY2lkIjoiMTA1MTA5MDU0OEY2QTUyQjlFM0JCODkwRjYwQUVGMEIifQ==
#### Linked PRs in other repos
https://github.com/PeerioTechnologies/peerio-icebear/pull/81
#### Manual testing instructions  
1. go to files
2. select file to move
3. search field should be visible in popup
4. entering a search term should trigger filtering
#### Additional notes
no 

----
### Reviewer

* Localisation.
* Architectural decisions.
* Performance.
* Comments.
* Big feature tech design documentation.
* Readability.
* DRY.
* Can we extract something to SDK?
* Are there any bicycles invented that should be replaced with existing SDK
* Are there unit and e2e tests added?

----
### Repository owner
